Docs/Configuration.tex

```diff
@@ -3856,10 +3886,12 @@ \subsection{Security Properties}\label{miscsecurityprops}
     scanning devices directly connected to PCI bus (e.g. VIRTIO).
   \end{itemize}
 
-  \emph{Note}: Given the above description, \texttt{0xF0103} value is expected to allow
-  scanning of SATA, SAS, SCSI, and NVMe devices with APFS file system, and prevent scanning
+  \emph{Note}: Given the above description, a value of \texttt{0xF0103} can be expected to allow
```

Comment:
There is no probability here. It should do this if working correctly. In my opinion, is expected is correct.

Reply:
Changed but note that "is expected" implies there is a probability, maybe small. but still present, that it may not happen.

If indeed there is no such probability, then should be `a value of \texttt{0xF0103} will allow`